web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -114,6 +114,14 @@ export const SidebarCardsList = (props: Props): ReactElement => {
     <>
       <h2 className="h1-styling margin-top-0">{Config.dashboardDefaults.sidebarHeading}</h2>
       <hr className="margin-top-2 margin-bottom-3 bg-cool-lighter" aria-hidden={true} />
+      {props.topCards.length === 0 && props.bottomCards.length === 0 && (
+        <div data-testid="empty-for-you-message" className="fdc fac">
+          <div className="text-center margin-bottom-3">
+            <Content>{Config.dashboardDefaults.emptyForYouMessage}</Content>
+          </div>
+          <img src={`img/for-you-section.svg`} alt={"For You Section"} />
```

Comment:
[boulder] I don't think "For You Section" is an accurate alt name for this image. A key finding of our accessibility audit thus far has been that we need our alt descriptions to be meaningful. If the image doesn't actually provide meaningful information it's actually better to hide this from the screen reader.

For example, if you had an image of a bar graph conveying a particular statistic you would set the alt text to be "graph illustrating covid cases over the last 6 months" (or whatever it is). In this case, providing an accurate alt description "woman pointing at vector images" doesn't add to the user experience. It's better to hide the image.

[Here's the item from Quality Logic talking about this.](https://www.pivotaltracker.com/story/show/185791759)

Reply:
Done!